pages/opportunities/jobs.md

```diff
@@ -22,8 +22,30 @@ permalink: /jobs/
 
 <br>
 
+{% assign board_size = site.data.related-jobs | size %}
```

Comment:
Done in [1724953](https://github.com/USRSE/usrse.github.io/pull/771/commits/17249532aaf92c9707dacb0a04987cbb21c045a1)
I think it's an improvement.

Reply:
Oh, and in case you're wondering - I did try to pass the filename as another parameter to the include in order to remove the duplicate `{% assign sorted_jobs` statements but I kept getting Liquid syntax errors and couldn't find the right way to get it to correctly evaluate in the template.

Reply:
@crd477 you don't need to pass the filename, you want to pass the variable, the list of sorted jobs you derived.

Reply:
and make the names unique between related and not. The reason is if the page is ever processed in parallel we don't want some kind of race condition having two lists of the same name going into one template. Unlikely, but it's more clear to be explicit with variables.

Reply:
> @crd477 you don't need to pass the filename, you want to pass the variable, the list of sorted jobs you derived.

Yes, it does this at least.

Reply:
Don't worry about it, I'll pull this tonight and test/work on it. Thanks!

Reply:
We aren't communicating well. I'm saying you should be passing the variable list of jobs as variable, not a filename. Don't worry about it, it will be faster for me to just do it.

Reply:
OK, sorry for all the hassle.